apps/ui/src/composables/useWeb3.ts

```diff
@@ -159,6 +149,34 @@ export function useWeb3() {
     }
   }
 
+  function attachProviderEvents(provider) {
+    const providerName: Connector = provider?.connectorName || 'injected';
+
+    if (loadedProviders.value.has(providerName)) return;
+    loadedProviders.value.add(providerName);
+
+    if (!provider.on || STARKNET_CONNECTORS.includes(providerName)) return;
```

Comment:
Looks like we reverted fix from https://github.com/snapshot-labs/sx-monorepo/pull/540 unintentionally as we now return before adding `accountsChanged` event.

Reply:
Any tips to avoid those merge conflict ? 9 out of 10 PR are getting merge conflicts